privacy.md

```diff
@@ -0,0 +1,128 @@
+
+### Our Commitment to Children's Online Privacy
+
+Out of special concern for children's privacy, the Cockpit Project does not knowingly accept online personal information from children under the age of 13. The Cockpit Project does not knowingly allow children under the age of 13 to become registered members of our sites. The Cockpit Project does not knowingly collect or solicit personal information about children under 13.
+
+In the event that the Cockpit Project ever decides to expand its intended site audience to include children under the age of 13, those specific web pages will, in accordance with the requirements of the Children's Online Privacy Protection Act (COPPA), be clearly identified and provide an explicit privacy notice addressed to children under 13. In addition, The Cockpit Project will provide an appropriate mechanism to obtain parental approval, allow parents to subsequently make changes to or request removal of their children's personal information, and provide access to any other information as required by law.
+
+### About Links to Other Sites
+
+This site contains links to other sites. the Cockpit Project does not control the information collection of sites that can be reached through links from [cockpit-project.org](/). If you have questions about the data collection procedures of linked sites, please contact these sites directly.
+
+### Your Rights and Choices in the EEA
+
+Where the EU General Data Protection Regulation 2016/679 (“GDPR”) applies to the processing of your personal data, especially when you access the website from a country in the European Economic Area (“EEA”), you have the following rights, subject to some limitations, against the Cockpit Project:
+
+* The right to access your personal data; 
+* The right to rectify the personal data we hold about you;
+* The right to erase your personal data;
+* The right to restrict our use of your personal data;
+* The right to object to our use of your personal data;
+* The right to receive your personal data in a usable electronic format and transmit it to a third party (also known as the right of data portability); and
+* The right to lodge a complaint with your local data protection authority. 
```

Comment:
This seems particularly hairy to me. cockpit-project.org doesn't store such data, so all of that would apply to e. g. GitHub or Red Hat mailing lists. We don't have direct control over this, so if a user wants their data changed or removed, we can only forward that to these sites, but cannot actually enforce this.